router/pods_test.go

```diff
@@ -800,30 +800,26 @@ func TestGetHealthCheckFromPodPort(t *testing.T) {
 
 // TestValidatePath tests the internal function to validate a proper path used in annotations
 func TestValidatePath(t *testing.T) {
-	testNoPrefix := "test"
-	testPathFail := "/test/%2a/%"
-	testPathPass := "/test/%2a/aa/a"
-	testPathPassProperEncoding := "/test/par%2ate/aa/a"
-	testPathFailInvalidEncoding := "/test/hello%zzworld/aa/a"
```

Comment:
But does it benefit the user to be able to specify a path that is invalid outside of nginx?  Doing so might end up with the path being inaccessible even if nginx will serve it.  The purpose of these checks were to tell the user they've defined an invalid URI path and I still think they would be useful checks on top of the nginx check.